docs/v1/openapi.yaml

```diff
@@ -535,6 +560,25 @@ components:
       required:
         - request_uri
         - expires_in
+    InitiateOIDCData:
+      title: InitiateOIDCData
+      type: object
+      properties:
+        presentation_definition_id:
+          type: string
+        purpose:
+          type: string
+    InitiateOIDCResponse:
```

Comment:
plz be more specific in naming those models as well - Initiate OIDC Issuance also has similar structures

Reply:
yes yes